site/docs/v1/tech/oauth/endpoints.adoc

```diff
@@ -786,6 +786,15 @@ The refresh token as described by https://tools.ietf.org/html/rfc6749#section-1.
 This field will be returned in the response when `offline_access` scope was requested unless refresh tokens have been disabled for this application.
 See `oauthConfiguration.generateRefreshTokens` in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth UI when editing an application.
 
+[field]#refresh_token_id# [type]#[UUID]#::
+The unique ID of the refresh token. This ID can be used to retrieve or revoke a specific refresh token when using the link:/docs/v1/tech/apis/jwt[Refresh Token API].
++
+This field will be returned in the response when `offline_access` scope was requested unless refresh tokens have been disabled for this application.
+See `oauthConfiguration.generateRefreshTokens` in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth UI when editing an application.
```

Comment:
```suggestion
Whether a refresh token is allowed for an application can be controlled by the   `oauthConfiguration.generateRefreshTokens` field in the link:/docs/v1/tech/apis/applications[Application API] or the `Generate refresh tokens` toggle in the FusionAuth admin UI.
```

Reply:
This was copied from the `refresh_token` description. Do you want this suggested copy used for both?

Reply:
Yes please.